### DISCUSSION TITLE
version 3.58.0 doesn't supporto macOS Mojave

### DISCUSSION THREAD

**Discussion by kintaro1981** [mifi/lossless-cut#1761] [General]
I tried to install it but I can't run it.
Is it possible make it support macOS Mojave? (3.56.0 support it)

**Comment by mifi** [comment]
See https://github.com/mifi/lossless-cut/blob/master/requirements.md

**Comment by mifi** [comment]
It’s possible but would take a lot of work

**Reply by kintaro1981** [reply]
I understand
How about a double build? Too much work too? (Or maybe a double
Build only “important/major” release?)

for now I’m going to stay on 3.56.0 that is working ok for what I need to do.

thank you for what you have done so far.

**Reply by mifi** [reply]
unfortunately double builds complicate software development too much
